pgctl/config.py

```diff
@@ -22,7 +22,7 @@
     from yaml import load as yaml_load
 except ImportError:  # pragma: no cover
     def yaml_load(dummy_file):
-        pass
+        del dummy_file
```

Comment:
i'd rather see a pylint ignore comment, i'd go "wtf" and delete this if i came across it